pallets/parachain-staking/src/lib.rs

```diff
@@ -47,9 +47,11 @@
 //! To leave the set of delegators and revoke all delegations, call `leave_delegators`.
 
 #![cfg_attr(not(feature = "std"), no_std)]
+#![allow(deprecated)]
```

Comment:
A comment about this might help, as it could silence valid warnings

Reply:
Oops that was only for dev. Removing.